dcos/jsonitem.py

```diff
@@ -283,8 +283,9 @@ def _parse_url(value):
     if value_regex is None:
         scheme_match = re.match(scheme_pattern, value, re.IGNORECASE)
         if scheme_match is None:
-            msg = 'Please check url {!r}. Missing http(s)://'.format(value)
-            raise DCOSException(msg)
+            logger.debug("Defaulting URL to https scheme")
+            value = "https://" + value.lstrip(scheme_pattern)
```

Comment:
`value.lstrip(scheme_pattern)` is a funny bug! 

It will remove all characters contained in the character set defined by `scheme_pattern` from the trailing part of the partial URL:
```>>> set('^(?P<scheme>(?:(?:https?)://))')
{'<', 'p', 'h', 'e', ')', '/', 's', '(', '^', 'P', '?', '>', ':', 'c', 'm', 't'}
>>> value = 'powerdcos.com'
>>> scheme_pattern = r'^(?P<scheme>(?:(?:https?)://))'
>>> "https://" + value.lstrip(scheme_pattern)
'https://owerdcos.com'
```